basedtyping/__init__.py

```diff
@@ -347,3 +350,17 @@ def issubform(
     if forminfo is Never:
         return False
     return issubclass(form, forminfo)  # type: ignore[arg-type]
+
+
+if TYPE_CHECKING:
+
+    Untyped: TypeAlias = Any  # type: ignore[no-any-explicit]
```

Comment:
maybe add a comment that says basedmypy doesnt actually look at this

Reply:
done

Reply:
wheres that comment i dont see it

Reply:
[is this it?](https://github.com/KotlinIsland/basedtyping/pull/44/files#diff-7fd05312aa4b91c50fc6766fd2789845da2abc401733491608cca183e50ea6a8R379)

Reply:
The comment should clarify that basedmypy will still treat it as the special form.